ui/src/components/build/BuildPipelineStage.js

```diff
@@ -1,10 +1,24 @@
 import React, { Component } from 'react';
+import Moment from "react-moment";
+
+import BuildStatus from '../common/BuildStatus';
+
+import "./BuildPipelineStage.css";
 
 class BuildPipelineStage extends Component {
+    // TODO: add webhook or poll end point to set pipeline stages
```

Comment:
is there a special endpoint we need to get the pipeline info, or is the current builds/buildconfigs endpoint is enough?

Reply:
The json object returned from `/builds` contain a `phase` which appears to be the right information. Maybe @JameelB can add some insight into what they used previously.

Reply:
@wei-lee @ciaranRoche There's an [annotation](https://github.com/aerogear/origin-web-console/blob/aerogear-mcp/app/scripts/directives/buildPipeline.js#L25) in the build object called `jenkinsStatus` which contains information on the pipeline stages. This is then used for rendering the [pipeline info](https://github.com/aerogear/origin-web-console/blob/aerogear-mcp/app/views/directives/_build-pipeline-expanded.html#L32-L45) on the build page.

Reply:
Awesome, thanks @JameelB